charts/memgraph/values.yaml

```diff
@@ -4,27 +4,71 @@ image:
   tag: ""
   pullPolicy: IfNotPresent
 
-# Can be increased to more but there is no replication or HA support in this chart.
+## Image Secrets for custom images
+# imagePullSecrets: [{ name: regcred }]
+
 replicaCount: 1
 
+## Node Affinity Preferred
+# By setting theses parameters the PREFERRED deployment will be done first on the match LABELS with key and value then on other nodes.
+# nodeKey: "nodegroup" give the name of a key
+# Operator is In
+# nodeValue: "memgraph" give the value of the key
+affinity:
+  nodeKey:
+  nodeValue:
+
 service:
-  type: NodePort
+  ## ClusterIP, NodePort, LoadBalancer
+  # ClusterIP keep the service inside the cluster makes it secure
+  # NodePort would create a external port change port: between 30000-32767 accessible to all the nodes and Public IPs if not in a VPC
+  # LoadBalancer is compabile with Cloud Providers on port: 80 without SSL redirected to the 7687
+  type: ClusterIP
   port: 7687
   targetPort: 7687
   protocol: TCP
   annotations: {}
-
+  
 persistentVolumeClaim:
-  storagePVCClassName: ""
-  storagePVC: true
-  storagePVCSize: 1Gi
-  logPVCClassName: ""
-  logPVC: true
-  logPVCSize: 256Mi
+  ## createStoragePVC `true` will create for each statefulset server a Persistant Volume Claim
+  ## `false` will let you choose an existing Persistant Volume Claim or will create one with an existing volume
+  createStorageClaim: true
+  ## Using a Storage Class Name with policy `retain` will keep the Persistant Volume Claim and the Volume
+  ## If you use a Storage Class Name with policy `delete` the Persistant Volume Claim and Volume will be deleted when the helm release is deleted
+  storageClassName: ""
+  ## Storage Size must me at minimum 4x the maximum size of your Dataset for Snapshots 
+  ## See documentation for choosing the right size depending on the number of Snapshots you want to keep
+  ## Default is 3 snapshots and you need space to create a new one and WAL files
+  storageSize: 500Gi
```

Comment:
I understand you have a bigger scale, but let's keep the default to something more reasonable so that someone can start on their laptop. I think 1GBi is sufficient for this.

Reply:
Of course, since there are no DB of 1GB and Helm Chart are mostly used for STAGING or PROD I recommend a generic value like 10Gi as the start-up of Memgraph will create almost 500Mb of data already.